framework/application/controllers/generic/GenericController.ts

```diff
@@ -4,12 +4,13 @@ import ResponseRequest from '../../../crosscutting/util/ResponseRequest';
 import IGeneric from '../../../domain/generics/IGeneric';
 
 export default abstract class GenericController<T> implements IGenericController {
-  public _service: IGeneric<T>;
+  public nameService:string;
 
-  constructor(service: IGeneric<T>) {
-    this._service = service;
+  constructor(nameService: string) {
```

Comment:
Qual a razão de recebermos apenas um serviço aqui? Tenho a impressão que as vezes vai fazer sentido que isso seja utilizado com arguments https://stackoverflow.com/questions/2141520/javascript-variable-number-of-arguments-to-function

Reply:
Da forma que ta , um controller chama apenas um serviço, se algum serviço precisar chamar outro ai ele recebe o mesmo no construtor como injeção de dependência..